action.yml

```diff
@@ -1,23 +1,37 @@
-name: "Moropo - Trigger Mobile App Test Run"
-author: "Moropo"
-description: "Uses Moropo.com to execute a mobile app UI test"
+name: 'Moropo - Trigger Mobile App Test Run'
+author: 'Moropo'
+description: 'Uses Moropo.com to execute a mobile app UI test'
 inputs:
   scheduled_test_id:
-    description: "Moropo Scheduled Test ID - find this in your scheduled test setup wizard"
+    description: 'Moropo Scheduled Test ID - find this in your scheduled test setup wizard'
     required: true
   api_key:
-    description: "Moropo Api Key - find this in your app settings in the Moropo dashboard"
+    description: 'Moropo Api Key - find this in your app settings in the Moropo dashboard'
     required: true
   expo_release_channel:
-    description: "Expo Release Channel (optional) - Pass this release channel to your test run"
+    description: 'Expo Release Channel (optional) - Pass this release channel to your test run'
     required: false
   github_token:
-    description: "GitHub access token (optional) - so that the action can comment on the PR with the test trigger feedback"
+    description: 'GitHub access token (optional) - so that the action can comment on the PR with the test trigger feedback'
     required: false
     default: ${{ github.token }}
+  build_path:
+    description: 'Path to the build artifact to upload to Moropo before running the test'
+    required: false
+  upload_build:
```

Comment:
Is this meaningful? passing in a build path should be enough to infer that we want to upload it